deployment/Dockerfile

```diff
@@ -1,5 +1,20 @@
 FROM bde2020/spark-master:2.3.1-hadoop2.7
 
+COPY sources.list /etc/apt
+RUN \
+  rm /etc/apt/sources.list.d/* \
+  && apt-get update \
+  && apt-get upgrade -y \
+  && apt-get clean
+
+RUN apt-get dist-upgrade -y
+
+RUN \
+  apt-get install -y --no-install-recommends postgresql-client \
+  && rm -rf /var/lib/apt/lists/*
+
 COPY osmesa-apps.jar /opt/osmesa-apps.jar
 COPY streaming/log4j.properties /spark/conf/
+COPY refresh-views.sh /usr/local/bin/refresh-views.sh
```

Comment:
It looks like with these changes that we've got one container image pulling double duty for two entirely separate tasks. 

As far as I can tell, the image running refresh-views could just be a very small base alpine linux image with psql package installed. If we moved it to a separate image we'd also presumably get faster start times when we trigger these tasks since the AWS executor would be downloading much smaller images to run. 

Can you create a follow up issue to split this Dockerfile in two?